.github/workflows/cleanup.yml

```diff
@@ -109,8 +108,13 @@ jobs:
       - name: 'Set up Cloud SDK for tool request'
         if: '${{ hashFiles(''tool.yaml'') != '''' }}'
         uses: 'google-github-actions/setup-gcloud@e30db14379863a8c79331b04a9969f4c1e225e0b' # ratchet:google-github-actions/setup-gcloud@v1
-      - name: 'Install AOD CLI'
-        run: 'go install github.com/abcxyz/access-on-demand/cmd/aod@${{ inputs.aod_cli_version }}'
+      - name: 'Setup AOD'
+        uses: 'abcxyz/pkg/.github/actions/setup-binary@main' # ratchet:exclude
```

Comment:
You should pin this to a SHA, throughout.

Since we are serving this workflow, some customers might required these are pinned

Reply:
Got it, updating.

Reply:
Done, thx!